include/SFML/Graphics/Font.hpp

```diff
@@ -364,22 +364,15 @@ class SFML_GRAPHICS_API Font
         std::vector<Row> rows;    //!< List containing the position of all the existing rows
     };
 
+    using PageList = std::list<Page>; //!< List of pages, where each page corresponds to a texture
+    using PageListTable = std::unordered_map<unsigned int, PageList>; //!< Table mapping a character size to its list of pages (textures)
```

Comment:
I can't say the original reasoning why, but not sure it hugely matters - I presume it was to avoid reallocations. If you think there's a better container to use I'm all ears

Reply:
I think it was rather to ensure that pointers to elements remain valid even if new elements are inserted or existing ones removed.

If this happens only at the end or beginning, `std::deque` would also work -- in any case, let's not change this within this PR, too much risk of breaking 🙂